npm-debug.log

```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok

```

Comment:
Please remove npm-debug.log, this file is not necessary and is automatically generated by npm when something went wrong.